virtool/startup.py

```diff
@@ -275,7 +273,9 @@ async def startup_postgres(app: Application):
 
     logger.info("Connecting to PostgreSQL")
 
-    app["pg"] = await virtool.pg.utils.connect(postgres_connection_string)
+    pg = await virtool.pg.utils.connect(postgres_connection_string)
+
+    return pg
```

Comment:
* Just call `connect()` directly from `startup_databases()` and get rid of `startup_postgres()`.
* Move logging to `connect()`.